src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -73,7 +73,7 @@ public function execute()
                     $func = 'ldap_search';
                     break;
                 default:
-                    throw new LdapException(sprintf('Could not search in scope %s', $this->options['scopen']));
+                    throw new LdapException(sprintf('Could not search in scope %s', $this->options['scope']));
```

Comment:
would you ind adding double quotes around the "%s" please?

Reply:
Done.